_data/tagging-status.yml

```diff
@@ -12489,6 +12497,51 @@
   tasks: needs tests
   updated: 2024-07-17
 
+- name: extarticle
+  type: class
+  status: unknown
+  issues:
+  tests: false
+  tasks: needs tests
+  comments: "Perhaps safer to use article + extsizes in the interim."
```

Comment:
Is this comment necessary? Anything from article.cls that is incompatible with tagging is redefined after the class is loaded, so I think the same should be true for extarticle

Reply:
Oh I see https://github.com/latex3/tagging-project/discussions/939 now. The comments should say basically what Ulrike said in the discussion so that looking back one can see what needs fixing